rewrite-groovy/src/main/java/org/openrewrite/groovy/GroovyParserVisitor.java

```diff
@@ -599,6 +621,11 @@ varargs, emptyList(),
                     Markers.EMPTY
             );
 
+            if (isConstructorOfInnerNonStaticClass) {
+                ((BlockStatement) method.getCode()).getStatements().remove(0);
+                ((BlockStatement) method.getCode()).getStatements().remove(0);
```

Comment:
Should we do this neater by making a copy except the first two items? Or is this fine enough?